### PR TITLE
Add Vagrantfile

### PR DESCRIPTION
**Pull request by sufuf3**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    sudo tar -zxf go1.10.2.linux-amd64.tar.gz -C /usr/local/
+    echo 'export GOROOT=/usr/local/go' >>  /home/$USER/.bashrc
+    echo 'export GOPATH=$HOME/go' >> /home/$USER/.bashrc
+    echo 'export PATH=/home/$USER/protoc/bin:$PATH:$GOROOT/bin:$GOPATH/bin' >> /home/$USER/.bashrc
+    export GOROOT=/usr/local/go
+    export GOPATH=$HOME/go
+    export PATH=/home/$USER/protoc/bin:$PATH:$GOROOT/bin:$GOPATH/bin
+    # setup golang dir
+    mkdir -p /home/$USER/go/src
+    rm -rf /home/$USER/go1.9.1.linux-amd64.tar.gz
```
Comment (hwchiu) [inline Vagrantfile]: Please change to `go1.10.2.linux-amd64.tar.gz`

**Comment by hwchiu** [comment]
@John-Lin @yarencheng @chenyunchen  this vagrant is used for setup a friendly develop environment for vortex.
We will setup a single-host ubuntu VM with k8s cluster and openvswitch.

**Comment by hwchiu** [comment]
@WakeupTsai  also.
In the next patch, we will add a README to instruct how to 
1. Use the host's kubectl to control the vagrant's k8s cluster
2. Use the host's docker clit to control the vagrant's docker daemon

**Comment by hwchiu** [comment]
Fix the directory name `developerss`

**Comment by sufuf3** [comment]
Sorry about that.
I fixed it.

**Pull request merged**